spec/basic/processedconfig_spec.js

```diff
@@ -0,0 +1,11 @@
+var util = require('util');
+
+describe('configuration elements', function() {
+  it('should have access to the processed config block', function() {
+    browser.getProcessedConfig().then(function(config) {
+      expect(config.capabilities.browserName).toEqual('chrome');
+      expect(config.capabilities.version).toEqual('ANY');
+      expect(config.params.login.name).toEqual('Jane');
```

Comment:
Also a quick note - the test will not wait for this 'then' block (and so not execute these tests) since it's a q promise and not a webdriver promise. I'm resolving this by making `getProcessedConfig` return a webdriver promise, so that it will automatically be waited for.